### PR TITLE
ISPN-14057 Toolbar in Protobuf schema

### PR DESCRIPTION
**Pull request by pnaik1**
This PR will resolve https://issues.redhat.com/browse/ISPN-14057

Please review my changes. :)

### CODE REVIEW

File: src/app/ProtoSchema/ProtobufSchemasDisplay.tsx [hunk 1]
```diff
@@ -270,7 +281,10 @@ const ProtobufSchemasDisplay = (props: {
             >
               <DataListItemRow>
                 <DataListToggle
-                  onClick={() => toggle(protoSchema.name)}
+                  onClick={() => {
+                    toggle(protoSchema.name);
+                    console.log(protoSchema.name);
```
Comment (dpanshug) [inline src/app/ProtoSchema/ProtobufSchemasDisplay.tsx:286]: Remove console statement

**Review by dpanshug — CHANGES_REQUESTED** [review]
Hi @pnaik1, Thanks for contributing

File: src/app/ProtoSchema/ProtobufSchemasDisplay.tsx [hunk 2]
```diff
@@ -290,6 +304,11 @@ const ProtobufSchemasDisplay = (props: {
                     >
                       {displayProtoError(protoSchema.error)}
                     </DataListCell>,
+                    <DataListCell width={2}>
+                      {expanded.includes(protoSchema.name)
+                        ? buildSchemaToolbar(protoSchema.name)
+                        : ''}
+                    </DataListCell>,
```
Comment (dpanshug) [inline src/app/ProtoSchema/ProtobufSchemasDisplay.tsx:311]: Remove `,`

**Review by dpanshug — APPROVED** [review]

**Comment by dpanshug** [comment]
The build is failing, need to check second commit [Merge branch 'infinispan:main' into changes](https://github.com/infinispan/infinispan-console/pull/261/commits/fd1403366a5754c8741db4550639d5debb3665c5)

**Comment by andyuk1986** [comment]
I am looking into failing test.

**Review by andyuk1986 — APPROVED** [review]
Tests fixed and passed.

**Pull request merged**